tools/test/generateAllScreenshots.py

```diff
@@ -117,11 +119,11 @@ def generateJavascriptFile():
     data = [["en", "en-dark"] + languages]
     files = sorted(
         os.listdir("tests/uitests/src/test/snapshots/images/"),
-        key=lambda file: file[file.find("_", 6):],
+        key=lambda file: file,
```

Comment:
The idea was to sort the gallery by View name, so for instance:
`ui_T_t[f.preferences.impl.about_AboutView_null_AboutView-Day-0_1_null_0,NEXUS_5,1.0,de].png` ->
`AboutView-Day-0_1_null_0,NEXUS_5,1.0,de].png`

(note sure about the `6` :/)

With the new name of screenshot file it should be:
`features.preferences.impl.about_AboutView_Day_0_de.png` ->
`features.preferences.impl.about_AboutView_Day_0_de.png`

I think now the code should be

```suggestion
        key=lambda file: file[file.find("_", 1):],
```

But please run `./tools/test/generateAllScreenshots.py` to check the change on `data.js`.